buttons-examples/iOS/ios-FAB.md

```diff
@@ -152,80 +152,204 @@ in the spec.
     (representing a day of the week) has the `accessibilityHint` value, "Toggles
     this day."
 
+## Regular FABs
 
-## Floating action button
+Regular FABs are FABs that are not expanded and are a regular size.
 
-You can apply a theme to the FAB using `Themes`.
+### Regular FABs example
 
-Source Code APIs:
+To create a regular FAB use the `+floatingButtonWithShape:` constructor with a value of `MDCFloatingButtonShapeDefault` and make sure the `mode` property is set to `MDCFloatingButtonModeNormal`.
 
-* MDCButton  (a subclass of [UIButton](https://developer.apple.com/documentation/uikit/uibutton))
-    * [Class description](https://material.io/develop/ios/components/buttons/api-docs/Classes/MDCButton.html)
-    * [GitHub source](https://github.com/material-components/material-components-ios/blob/develop/components/Buttons/src/MDCButton.h)
-* [Themes class description](https://material.io/develop/ios/components/theming/)  <!-- This is slated to be deprected, though the examples from https://material.io/develop/ios/components/buttons/api-docs/Classes/MDCButton.html appear to use this class--> 
+For more information on theming FABs see the [Theming section](#theming). 
 
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+let fab = MDCFloatingButton(shape: `default`)
+```
 
-<!-- To do:
```

Comment:
Do you have time for any FAB screenshots?

Reply:
I added 1 screenshot per type, let me know if I should do more. Oh, also I'll do some Shrine ones in the theming section like I did for Buttons.